app_to_web.md

```diff
@@ -66,6 +66,17 @@ Trigger registrations will accept a new response header as well:
 Attribution-Reporting-Register-OS-Trigger: "https://adtech.example/register", "https://other-adtech.example/register"
 ```
 
+The reporting origin can also optionally respond with a [dictionary structured header](https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-header-structure-15#section-3.2)
+`Attribution-Reporting-Info` to specify the preferred platform. The key is
+`preferred-platform` and the value is a [structured header token](https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-header-structure-15#section-3.3.4)
```

Comment:
Can we avoid the draft links and instead use the same links from our spec, ie.. https://httpwg.org/specs/rfc8941.html#dictionary ?

Reply:
Updated, thanks.